website/content/docs/getting-started/connect-to-target.mdx

```diff
@@ -38,51 +38,57 @@ within the authorized session. When you are finished making connections, simply
 It can be annoying to keep accepting host SSH key prompts as the port changes,
 much less having to keep copying and pasting the current port. To make this
 easier, Boundary includes connect helpers that take care of this work for you.
-These take the form of `boundary connect <subcommand>`. In the following
-example, the helper will automatically execute `ssh` for you, filling in the
-local address/port, and setting an expected host ID so that future connections
-on different automatically-allocated ports don't complain about the host ID
-changing (you'll still need to accept a host key the first time):
+These are written as `boundary connect <subcommand>`. In the following example,
+the helper will automatically execute `ssh` for you, filling in the local
+address/port, and setting an expected host ID so that future connections on
+different automatically-allocated ports don't complain about the host ID
+changing (you'll still need to accept a host key the first time).
 
 ```
 $ boundary connect ssh -target-id ttcp_1234567890
 ```
 
-If you want to pass additional flags to the SSH client, you can do so by adding
-them to the command line separated by a double-dash; anything after the double
-dash will be passed to the executed client. For instance:
+If you want to pass additional flags to the SSH client, add them to the command
+line separated by a double-dash; anything after the double dash will be passed
+to the executed client. For instance:
 
 ```
 $ boundary connect ssh -target-id ttcp_1234567890 -- -l some-other-user
 ```
 
 There is also a `-style` flag to allow the command to format arguments in a
-different style expected by different SSH clients. At the moment, besides `ssh`
-(the default), the `boundary connect ssh` command supports `-style putty` to
-support passing connection information to PuTTY.
+different style expected by different SSH clients. Besides the default `ssh`,
+the `boundary connect ssh` command supports `-style putty` to support passing
+connection information to PuTTY.
 
 One advantage to styles is that Boundary will provide information to the client
-in the format that makes sense. For example, if you want to specify a username
-other than your currently logged-in user, you can do so via the `-username`
-flag. This will ensure that regardless of whether you use the default `ssh`
-style or the `putty` style, the username is properly passed to the executed
-client -- you don't need to figure out the syntax yourself.
+in the format that makes sense. For example, the `-username` flag is used to
+specify a username other than your currently logged-in user. This ensures that
+regardless of whether you use the default `ssh` style or the `putty` style, the
```

Comment:
We've already described the available styles above, we could make this bit a little lighter.

```suggestion
regardless of your `-style` choice, the
```